ros2_foxglove_bridge/include/foxglove_bridge/param_utils.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <regex>
+#include <string>
+#include <vector>
+
+#include <rclcpp/node.hpp>
+
+namespace foxglove_bridge {
+
+constexpr char PARAM_PORT[] = "port";
+constexpr char PARAM_ADDRESS[] = "address";
+constexpr char PARAM_USETLS[] = "tls";
+constexpr char PARAM_CERTFILE[] = "certfile";
+constexpr char PARAM_KEYFILE[] = "keyfile";
+constexpr char PARAM_MAX_QOS_DEPTH[] = "max_qos_depth";
+constexpr char PARAM_TOPIC_WHITELIST[] = "topic_whitelist";
+constexpr char PARAM_BEST_EFFORT_TOPICS[] = "best_effort_topics";
```

Comment:
Although we should support both datatypes and topics at some point, datatypes will be a more common way to think about this I'm guessing since you generally want all of your sensor messages to be best-effort (for visualization at least), so all of the camera images and point clouds, but without having to name for example all six surround cameras and three lidars.

Reply:
Should I add that? 

Usually these heavy topics have a common suffix, e.g. camera and point cloud topics. For instance, for the realsense bagfile I used the following patterns: `--ros-args -p best_effort_topics:=[".*points",".*image_raw"]`

Reply:
I think we should get to a place where this works out of the box for 99% of users without any custom configuration. In every example of poor streaming performance so far, the heavy topics have one of three datatypes: sensor_msgs/Image, sensor_msgs/CompressedImage, sensor_msgs/PointCloud2. If we're making the assertion that recording the correct QoS profile and using it for playback is broken and the correct fix for that is a workaround in foxglove_bridge, let's make that fix work out of the box by forcing best-effort on those three datatypes by default.

If we want to provide topic regex matching for power users, that's fine, but I don't see it as a high-priority requirement.